tables/subject_history/ikemura21-pth.csv

```diff
@@ -0,0 +1,7 @@
+,ref_name,subject_name,event,event_date_cmp,event_date,location,iso_name,vaccine_name,severity
+0,Ikemura21,B.1.1.7_infection,isolation,~,2021-01-31,Japan,NULL,NULL,NULL
```

Comment:
This should be `infection` event, not `isolation` event

Reply:
I have an `isolation` event for `subject_name = B.1.1.7_infection` in the second entry, should both say `isolation`?
 Also, would the same apply to the 3rd and 4th entries?

Reply:
> I have an `isolation` event for `subject_name = B.1.1.7_infection` in the second entry, should both say `isolation`? Also, would the same apply to the 3rd and 4th entries?

For convalescent plasma/serum, the events in order are 1) infection then 2) isolation. 
For vaccinee plasma, the events in order are 1) 1st dose 2) 2nd dose 3) 2nd dose isolation. If there's booster dose then 3) would be 3rd dose 4) would be 3rd dose isolation
For infection then vaccination, the events in order are 1) infection 2) 1st dose 3) 2nd dose 4) 2nd dose isolation
For breakthrough infection, the events in order are 1) 1st dose 2) 2nd dose 3) infection 4) isolation.

So in the current study, the 1st and 3rd entry may change to `infection`, and the 2nd and 4th entries are still `isolation`.